### DISCUSSION TITLE
Theme Overlay Material Toolbar

### DISCUSSION THREAD

**Discussion by beppo-ivel** [appcelerator/titanium_mobile#13198] [Q&A] [answered]
If I click the 3 dots in the toolbar the menu-items pop up. And I want to change their text color. But I can't find the correct property

platform/android/res/values/myTheme.xml:
```
...
<style name="MyThemeOverlay_Toolbar" parent="@style/ThemeOverlay.MaterialComponents.Toolbar.Primary">
    <item name="actionMenuTextColor">#48a697</item>
    <item name="android:actionMenuTextColor">#48a697</item>
...
```
Also I can't find property to set the background of the popup menu to transparent.

**Comment by m1ga** [comment, marked as answer]
You really need to update your google-game 🤣 

Colored dots: https://stackoverflow.com/a/35933013
Menu background: https://stackoverflow.com/a/28979152
font color: https://stackoverflow.com/a/19212678

It took me more time to create the Titanium code for the menu 😄 

Full XML example:
```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>

<style name="defaultTheme" parent="Theme.MaterialComponents.DayNight">
	<item name="android:actionOverflowButtonStyle">@style/MyCustomTheme</item>
	<item name="actionOverflowButtonStyle">@style/MyCustomTheme</item>

	<item name="actionOverflowMenuStyle">@style/menubackground</item>

	<item name="android:itemTextAppearance">@style/TextAppearance</item>
</style>

<style name="MyCustomTheme" parent="@style/Widget.AppCompat.ActionButton.Overflow">
	<item name="android:tint">#f00</item>
</style>

<style name="menubackground" parent="Widget.AppCompat.PopupMenu.Overflow">
     <item name="android:popupBackground">@android:color/transparent</item>
</style>

<style name="TextAppearance">
    <item name="android:textColor">#0ff</item>
</style>

</resources>
```
will show red dots, aqua text in the popup and a transparent popup menu.

![Screenshot_1638057174](https://user-images.githubusercontent.com/4334997/143723774-690a7be9-55f5-4ac8-b136-101fb2c524de.png)

**Reply by beppo-ivel** [reply]
> You really need to update your google-game rofl

Tell me how ;) Can't resolve the pattern behind those naming conventions. It looks like I have to google for random attributes, which contains by random parent classes. For instance why there is no property for item-text-color in `ThemeOverlay.MaterialComponents.Toolbar.Primary`? Or why there is no `ThemeOverlay.MaterialComponents.PopupMenu.Overflow` or `ThemeOverlay.MaterialComponents.Toolbar.PopupMenu` ?

Also if I set `<item name="android:background">#0f34</item>` then this style affects the `<item name="android:popupBackground">#0342</item>` as well.

**Reply by m1ga** [reply]
I've just searched for `android actionbar popup background color` or `android actionbar change dot color` and it will show Stackoverflow posts right away. Android Theming is always a "google around" topic for me :smile:
